opencti-platform/opencti-graphql/src/utils/markingDefinition-utils.js

```diff
@@ -0,0 +1,15 @@
+import * as R from 'ramda';
+import { getEntitiesMapFromCache } from '../database/cache';
+import { ENTITY_TYPE_MARKING_DEFINITION } from '../schema/stixMetaObject';
+import { SYSTEM_USER } from './access';
+
+export const cleanMarkings = async (context, values) => {
```

Comment:
It would be nice to have a test for this method.

Reply:
Yes, it will be done later :)